packages/firestore/test/unit/specs/write_spec.test.ts

```diff
@@ -1127,4 +1127,21 @@ describeSpec('Writes:', [], () => {
         hasPendingWrites: true
       });
   });
+
+  specTest('Write is sent by newly started client', ['multi-client'], () => {
```

Comment:
FWIW, I'm starting to wonder if it'd make sense to put all the multi-tab tests together (or maybe the primary transition ones at least) and have more contextual descriptions.  "Multi-tab: Primary-Transition: Write is sent by newly started client" or something...

I don't feel super strongly at the moment though.